modules.d/95resume/module-setup.sh

```diff
@@ -13,7 +13,7 @@ check() {
     # Only support resume if hibernation is currently on
```

Comment:
Just a nitpick: the comment is wrong after this change. It should be "Only support resume if no swap is mounted on a net device".

Reply:
Fixed